main/SQLHelper.cpp

```diff
@@ -233,6 +233,7 @@ const char* sqlCreateHardware =
 "[Name] VARCHAR(200) NOT NULL, "
 "[Enabled] INTEGER DEFAULT 1, "
 "[Type] INTEGER NOT NULL, "
+"[LogLevel] INTEGER default 7, " //LOG_NORM + LOG_STATUS + LOG_ERROR
```

Comment:
My suggestion would be to turn all bits on by default. Including the future once! So if we use LogLevel as a 16 bit (or 32, see previous comment on size) bit-wise structure, should we set the default value to 65535 (xFFFF)? This way all bits are **On**

Reply:
I also want to do more work on the log window